Modules/Loadable/Markups/Testing/Python/MarkupsCurveMeasurementsTest.py

```diff
@@ -57,8 +57,8 @@
 # Turn off curvature computation
 curveNode.GetMeasurement("curvature max").SetEnabled(False)
 curvePointData = curveNode.GetCurveWorld().GetPointData()
-if curvePointData.GetNumberOfArrays() != 1:
-    exceptionMessage = "Unexpected number of data arrays in curve: " + str(curvePointData.GetNumberOfArrays())
+if curvePointData.GetNumberOfArrays() != 4:
```

Comment:
Add array names. Maybe add a helper function that takes a list of array names and compares it with array names in a pointData object. That could allow a bit more thorough testing.